### PR TITLE
Change property name so that api key is correct

### PR DESCRIPTION
**Pull request by HarryBosworth**
Spring property sources favour environment variables over properties files leading to the property `api.key` being set to `${API_KEY}` instead of using the property file which set the property to `${INTERNAL_API_KEY}`. This PR renames the property to `internal.api.key` so it is populated from the correct environment variable.

### CODE REVIEW

**Review by divya051988 — APPROVED** [review]

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -10,7 +10,7 @@ management.endpoints.enabled-by-default=true
 management.endpoints.web.base-path=/officer-delta-processor
 management.endpoints.web.path-mapping.health=/healthCheck
 server.port=8080
-chs.api.key=${CHS_INTERNAL_API_KEY}
+chs.internal.api.key=${CHS_INTERNAL_API_KEY}
```
Comment (PhilipGlover) [inline src/main/resources/application.properties:13]: we need a corresponding amendment to the compose file in `docker-chs` yeah?

**Review by PhilipGlover — CHANGES_REQUESTED** [review]
`OfficerDeltaProcessorApplicationTests` fails "Error creating bean with name 'deltaProcessor'"

File: src/main/resources/application.properties (hunk 1 above)
Comment (PhilipGlover) [reply src/main/resources/application.properties:13]: done with https://github.com/companieshouse/docker-chs-development/pull/351

**Review by PhilipGlover — APPROVED** [review]
have run successfully, tests pass, looking good.

**Pull request merged**